python/py-pytest-subtests/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        pytest-dev pytest-subtests 0.4.0
+name                py-${github.project}
+revision            0
+categories-append   devel
+platforms           darwin
+supported_archs     noarch
+license             MIT
+fetch.type          git
```

Comment:
please take this from PyPI instead to avoid the `fetch.type git`

Reply:
Okay, but pretty sure that will break things.

Would you please step in and survey the breakage and offer a fix? There will be some complaint from the build command that it's not a valid git repo.

I could never figure that one out, so I just used a git clone to make this issue go away.

Reply:
Done. I predict a CI failure. Would you please take a look when it comes?

Reply:
you only removed `fetch.type git`, that will likely not work indeed. I suggested to take the file from PyPI, which *should* work correctly, but for that to happen you'll need to remove the `github` PG and associated stuff as well.

Reply:
I tried switching to PyPI before but it throws the same error. I'll amend to demonstrate…

Reply:
To the best of my knowledge, this is the last remaining major issue. I'm expecting a CI build failure using PyPI now…

Reply:
Looks like I was wrong about the PyPI source, and this all builds fine.